modules/vaos/app/services/vaos/v2/appointments_service.rb

```diff
@@ -39,7 +39,7 @@ def post_appointment(request_object_body)
         end
       end
 
-      def update_appointment(appt_id:, status:)
+      def update_appointment(appt_id, status)
```

Comment:
curious why you removed the keyword parameters

Reply:
For consistency atm.  I like the explicit field naming better, I'll get a ticket in to change it.